spec/uploaders/file_uploader_spec.rb

```diff
@@ -15,10 +15,6 @@
     end
   end
 
-  after do
```

Comment:
I thought this commit is important so I put it alone... maybe can be squashed together with the first commit.